cmd/podman/common/create.go

```diff
@@ -603,6 +603,14 @@ func DefineCreateFlags(cmd *cobra.Command, cf *ContainerCLIOpts) {
 	)
 	_ = cmd.RegisterFlagCompletionFunc(sdnotifyFlagName, AutocompleteSDNotify)
 
+	secretFlagName := "secret"
+	createFlags.StringArrayVar(
+		&cf.Secrets,
+		secretFlagName, []string{},
+		"Add secret to contianer",
+	)
+
+	_ = cmd.RegisterFlagCompletionFunc(secretFlagName, completion.AutocompleteDefault)
```

Comment:
The newline should be below and not above. Also you should change the autocomplete function. see other comment